src/Storage/Commands.Storage/Common/Cmdlet/SetAzureStorageServiceProperties.cs

```diff
@@ -0,0 +1,71 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.WindowsAzure.Commands.Storage.Common.Cmdlet
+{
+    using System;
+    using System.Management.Automation;
+    using System.Security.Permissions;
+    using StorageClient = WindowsAzure.Storage.Shared.Protocol;
+    using Microsoft.WindowsAzure.Commands.Storage.Model.ResourceModel;
+
+    /// <summary>
+    /// Modify Azure Storage service properties
+    /// </summary>
+    [Cmdlet(VerbsCommon.Set, StorageNouns.StorageServiceProperty, SupportsShouldProcess = true), OutputType(typeof(PSSeriviceProperties))]
```

Comment:
@blueww what do you think about changing the name of this cmdlet to `Update-*` instead of `Set-*`? It seems like when more parameters are added to this cmdlet, only the values of properties provided should be updated and the other values should remain the same.

The `Update` verb is usually reserved for PATCH commands, which is what I think you would want to do in the future with this cmdlet, and the `Set` verb is usually reserved for PUT commands. Let me know what you think.

Reply:
Has updated the cmdelts name to Update-AzureStorageServiceProperty, help and changelog also updated.